install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+# Installs teabot.service.template into /etc/systemd/system/teabot.service
+# and enables it to run at boot-time by default.
+#
+# Use the --fake argument in testing, to use a secondary server, and avoid the
+# need for real USB scales + temperature sensor.
+
+set -e
+
+cd `dirname $0`
+
+sudo pip install -r requirements.txt
+
+if [ -f teabot.service ]
+then
+    echo "Please delete teabot.service file before continuing."
+    exit 1
+fi
+
+if echo $* | grep -q -- --fake
+then
+    ARGS=" --fake"
+fi
+
+cat teabot.service.template \
+    | sed "s:{{PWD}}:$PWD:g" \
```

Comment:
$PWD is set to the current directory by the shell, whenever you cd anywhere. I will put it in a temporary variable if you can think of a better name for it.